src/lib_ccaligner/generate_approx_timestamp.cpp

```diff
@@ -19,6 +19,11 @@ void CurrentSub::printToSRT(const std::string& fileName, outputOptions printOpti
 {
     std::ofstream out;
     out.open(fileName, std::ofstream::app);
+    
+    if(!out.good())
+    {
+        LOG("Couldn't open file %s", fileName);
```

Comment:
If we can not open file, we should probable halt - no point in continuing. Also it would be helpful to display the reason why the file couldn't be opened.